test/666_abuse_of_start_test.sh

```diff
@@ -0,0 +1,21 @@
+#! /bin/bash
+
+. ./config.sh
+
+start_suite "Abuse of 'start' operation"
+
+weave_on $HOST1 launch
+
+proxy_start_container $HOST1 --name=c1
+proxy docker_on $HOST1 create --name=c2 $SMALL_IMAGE grep "^1$" /sys/class/net/ethwe/carrier
+# Now start c2 with a sneaky HostConfig
+curl -X POST -H Content-Type:application/json -d '{"NetworkMode": "container:c1"}' http://$HOST1:12375/v1.20/containers/c2/start
```

Comment:
I suggest introducing a `docker_remote_on` in config.sh, which does the `curl`ing and can be invoked with `proxy docker_remote_on`.